### PR TITLE
Patched Altinn.Common.AccessToken dependency

### PR DESCRIPTION
**Pull request by acn-sbuad**
## Description

Patched library with bug fix for excessive logging. 

## Related Issue(s)
- N/A

### CODE REVIEW

File: src/Altinn.Notifications/Altinn.Notifications.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Common.AccessToken" Version="3.0.3" />
+    <PackageReference Include="Altinn.Common.AccessToken" Version="4.1.0" />
```
Comment (SandGrainOne) [inline src/Altinn.Notifications/Altinn.Notifications.csproj:13]: Now that I understand a bit better what it is that is breaking Authorization I believe this should work. Notification doesn't have ABAC based Authorization as far as I can remember.

For application with ABAC based Authorization I hope updating PEP to 1.4.0-alpha3 will fix the issues we've been having. We can try this in Events if you want.

File: test/Altinn.Notifications.Tests/Altinn.Notifications.Tests.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Common.AccessToken" Version="3.0.3" />
+    <PackageReference Include="Altinn.Common.AccessToken" Version="4.1.0" />
```
Comment (SandGrainOne) [inline test/Altinn.Notifications.Tests/Altinn.Notifications.Tests.csproj:10]: This reference should be unnecessary.

**Review by SandGrainOne — APPROVED** [review]

File: src/Altinn.Notifications/Altinn.Notifications.csproj (hunk 1 above)
Comment (acn-sbuad) [reply src/Altinn.Notifications/Altinn.Notifications.csproj:13]: will test in events.

File: test/Altinn.Notifications.Tests/Altinn.Notifications.Tests.csproj (hunk 2 above)
Comment (acn-sbuad) [reply test/Altinn.Notifications.Tests/Altinn.Notifications.Tests.csproj:10]: A package downgrade is introduced if we don't have this explicitly. Will keep for now.

**Pull request merged**